test/suites-ostree/isolated-network.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
+Documentation       Tests related to MicroShift running in an isolated network
+
+Resource            ../resources/ostree-health.resource
+Resource            ../resources/microshift-process.resource
+Resource            ../resources/microshift-host.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           network
+
+
+*** Variables ***
+${USHIFT_HOST}      ${EMPTY}
+${LB_CONFIG}        assets/isolated-lb-service.yaml
+${LB_NSPACE}        openshift-ingress
+${LB_SRVNAME}       isolated-lb-service
+${LB_PORTNUM}       31111
+
+
+*** Test Cases ***
+Verify Healthy Then Clean And Reboot MicroShift
+    [Documentation]    Verify no Internet access, check MicroShift health, clean
+    ...    MicroShift data and verify health again after rebooting the host
```

Comment:
Test case name and documentation is almost exactly like the keywords in the test.
Maybe we could include "what" are we testing for the name, and "why" instead of "how" for the description?

```suggestion
Verify MicroShift Is Healthy Running In Isolated Network
    [Documentation]    Make sure that MicroShift running in isolated network is healthy
```